### ISSUE TITLE
License perms

### ISSUE THREAD

**Issue by ronaldpetty** [Parquery/icontract#254]
When installing via pip. The license is placed in /usr/local

```
ERROR: Could not install packages due to an OSError: [Errno 13] Permission denied: '/usr/local/LICENSE.txt'
Consider using the `--user` option or check the permissions.
```

Maybe consider rename for global installation too LICENSE.icontract.txt (or subdirectory). Just a suggestion, keep up the good work!

**Comment by mristin** [comment]
@ronaldpetty could you please elaborate? (Or do you have maybe a pointer to a page that explains the problem in more detail?)

**Comment by ronaldpetty** [comment]
@mristin sorry for the delay. I had ran `pip install icontract` on my Mac. It failed due to the license being written to /usr/local/LICENSE.txt, where the directory is owned by root. While I should have used a `--user` option to redirect locally, it seems odd that it would try to write it out to a generic location like `/user/local`.

**Comment by mristin** [comment]
@ronaldpetty I could not reproduce the issue. If you have a colleague with a Mac, could you try if it also fails on his/her computer?

**Comment by mristin** [comment]
I observe the following behavior on Windows. The `LICENSE.txt` is installed in `venv\Lib\site-packages\icontract-2.6.2.dist-info\LICENSE.txt`.

**Comment by mristin** [comment]
@thiagokokada I see that on https://github.com/NixOS/nixpkgs/pull/199341 you mention that icontract also puts a `README.rst` in the same directory in the build environment as pylddwrap -- is that something which needs fixing?

Is this issue maybe related? If so, any ideas how to fix it?

**Comment by thiagokokada** [comment]
> Is this issue maybe related? If so, any ideas how to fix it?

@mristin I think so.

One idea that I can think is removing this [line](https://github.com/Parquery/icontract/blob/master/setup.py#L63). But if this is not possible for some reason, maybe those data files could be moved instead?

BTW, keep in mind that pylddwrap has the same issue: https://github.com/Parquery/pylddwrap/blob/master/setup.py#L71

**Comment by mristin** [comment]
@ronaldpetty @thiagokokada Could you please confirm that #269 fixes the issue, and re-open it if the fix did not work for you?

Please apologize for such a long delay (almost a year 😢).

**Comment by ronaldpetty** [comment]
@mristin I believe it works (but to be clear, I am not certain if installing from the repo is exactly same as installing from packages). Thanks for the perseverance!

```
% pip3 install git+https://github.com/Parquery/icontract            
Collecting git+https://github.com/Parquery/icontract
  Cloning https://github.com/Parquery/icontract to /private/var/folders/c5/l52zshy12q1bsfhp_sbdtg5r0000gn/T/pip-req-build-j477mh_o
  Running command git clone --filter=blob:none --quiet https://github.com/Parquery/icontract /private/var/folders/c5/l52zshy12q1bsfhp_sbdtg5r0000gn/T/pip-req-build-j477mh_o
  Resolved https://github.com/Parquery/icontract to commit 497125373aa0cf1230503b13d988c6c0aab582db
  Preparing metadata (setup.py) ... done
Requirement already satisfied: asttokens<3,>=2 in ./Library/Python/3.11/lib/python/site-packages (from icontract==2.6.2) (2.2.1)
Collecting typing_extensions
  Using cached typing_extensions-4.8.0-py3-none-any.whl (31 kB)
Requirement already satisfied: six in ./Library/Python/3.11/lib/python/site-packages (from asttokens<3,>=2->icontract==2.6.2) (1.16.0)
Installing collected packages: typing_extensions, icontract
  DEPRECATION: icontract is being installed using the legacy 'setup.py install' method, because it does not have a 'pyproject.toml' and the 'wheel' package is not installed. pip 23.1 will enforce this behaviour change. A possible replacement is to enable the '--use-pep517' option. Discussion can be found at https://github.com/pypa/pip/issues/8559
  Running setup.py install for icontract ... done
Successfully installed icontract-2.6.2 typing_extensions-4.8.0

[notice] A new release of pip available: 22.3.1 -> 23.2.1
[notice] To update, run: pip3 install --upgrade pip
```

**Comment by mristin** [comment]
@ronaldpetty I have just released a patch version 2.6.3 with this issue (hopefully) fixed.
